plugins/utils/repeat.nix

```diff
@@ -0,0 +1,17 @@
+{
+  lib,
+  ...
+}:
+lib.nixvim.vim-plugin.mkVimPlugin {
+  name = "repeat";
+  originalName = "vim-repeat";
+  package = "vim-repeat";
+
+  maintainers = [ lib.nixvim.maintainers.refaelsh ];
+
+  settingsExample = {
+    settings =
+      {
+      };
+  };
```

Comment:
```suggestion
```

Reply:
Also, note that settingsExamples should just be the settings value, not the whole assignment :slightly_smiling_face:

Reply:
> Also, note that settingsExamples should just be the settings value, not the whole assignment 🙂

Understood.